cookbook/specs/spec_add_ethereum.json

```diff
@@ -1353,6 +1353,129 @@
                                 }
                             }
                         ]
+                    },
+                    {
+                        "enabled": true,
+                        "collection_data": {
+                            "api_interface": "jsonrpc",
+                            "internal_path": "",
+                            "type": "POST",
+                            "add_on": "bundler"
+                        },
+                        "apis": [
```

Comment:
I noticed that all the APIs are defined with `"deterministic": false`. Why? some of them are deterministic

Reply:
Sorry, not sure how I missed that. I'm fairly certain the `eth_sendUserOperation` and `eth_supportedEntryPoints` are non-deterministic, but the rest are definitely deterministic.